### ISSUE TITLE
Object names still using plural for operators

### ISSUE THREAD

**Issue by simon3z** [openshift-kni/cnf-features-deploy#12]
I think we switched (or we are in the process of switching) the image names from "operators" to "operator" but some of the objects created here still carry the plural (e.g. CatalogSource, etc.)

One is (but there are others):

```
apiVersion: operators.coreos.com/v1alpha1
kind: CatalogSource
metadata:
  name: performance-addon-operators-catalogsource
  namespace: openshift-marketplace
spec:
  displayName: Openshift Performance Addon Operators
  icon:
    base64data: ""
    mediatype: ""
  image: quay.io/openshift/performance-addon-operators-registry
  publisher: Red Hat
  sourceType: grpc
```

cc @MarSik  @fromanirh

**Comment by fromanirh** [comment]
resolved by https://github.com/openshift-kni/cnf-features-deploy/pull/13

**Comment by MarSik** [comment]
#13 was closed, but #28 fixed that instead
